snappy_wrappers/wrappers/ascat/alleleCounter/alleleCounter.sh

```diff
@@ -0,0 +1,337 @@
+#!/bin/bash
+# More safety, by turning some bugs into errors.
+set -o errexit -o pipefail -o noclobber -o nounset
+
+# =================================================================================================
+#
+# Mimicks C implementation of alleleCounter (https://github.com/cancerit/alleleCount)
+# which is required by ascat (https://github.com/VanLoo-lab/ascat), but not in bioconda nor conda-forge.
```

Comment:
I think we should just contribute a recipe for this to bioconda

Reply:
Yes, I agree. I'll try to volunteer that, but I don't want to support a software that 1) is poorly written in my opinion, 2) difficult to install & 3) is really an non-entity (bcftools + awk does the job nicely)

Reply:
I understand (+ agree)